modules/node/content/cms_node.e

```diff
@@ -78,6 +80,10 @@ feature -- Access
 		deferred
 		end
 
+	status: INTEGER
+			-- Associated status for the current node
+			-- [{1,Not_Published}, {2, Published}, {3, Trash}]	
```

Comment:
maybe reference the CMS_NODE_CONSTANTS interface or similar.

We could also start at '0' for Not Published, having the default value = 0 as being the default status as not published